toolkit/build.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
+
+TOBUILD=
+BID=lkg
+RC=n
+WCREFRESH=n
+SDIR=../SPECS
+USEPKGBLDCACHE=y
+
+helpFuncion () {
+	echo ""
+	echo "-h  : Print detailed usage"
+        echo "-s  : Spec Dir [Default SPECS] [Possible values: ../SPECS-EXTENDED, ../SPECS]"
+        echo "-b  : DAILY_BUILD_ID of the format of 3-0-YYYYMMDD [Default 'lkg']"
+        echo "-c  : Run with check distabled/enabled [Default RUN_CHECK=n]"
+        echo "-p  : Package to build/rebuild [Default all]"
+        echo "-r  : Refresh worker chroot [Default false]"
```

Comment:
I would invert `-r` to off by default. Most people won't be messing with the go tools (which is the main reason the chroot regenerates), and can lead to some weird inconsistencies if the toolchain packages get out of sync with the chroot.

Reply:
Sure